### PR TITLE
Added Frogbot support for viewing all of the vulnerabilities + Updated Frogbot images

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.

---
Fixes issue #96 
- The 'JF_INCLUDE_ALL_SCAN' environment variable now allows Frogbot to display all existing vulnerabilities, not just the new ones. 
- The Frogbot images have been updated.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -866,6 +866,7 @@ golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5/go.mod h1:yigFU9vqHzYiE8U
 golang.org/x/crypto v0.0.0-20190820162420-60c769a6c586/go.mod h1:yigFU9vqHzYiE8UmvKecakEJjdnWj3jj499lnFckfCI=
 golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392/go.mod h1:/lpIB1dKB+9EgE3H3cr1v9wB50oz8l4C4h62xy7jSTY=
 golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550/go.mod h1:yigFU9vqHzYiE8UmvKecakEJjdnWj3jj499lnFckfCI=
+golang.org/x/crypto v0.0.0-20200604202706-70a84ac30bf9/go.mod h1:LzIPMQfyMNhhGPhUkYOs5KpL4U8rLKemX1yGLhDgUto=
```
Comment (talarian1) [inline go.sum:869]: Unnecessary change to go.sum
You may run `go mod tidy` before committing changes to go.mod & go.sum.

**Review by talarian1 — CHANGES_REQUESTED** [review]

File: commands/utils/params.go [hunk 2]
```diff
@@ -16,6 +16,7 @@ type FrogbotParams struct {
 	GitParam
 	WorkingDirectory   string
 	InstallCommandName string
+	IncludeAllScan     string
```
[the remark this answers is not in the document]
Comment (omerzi) [reply commands/utils/params.go]: It represent an environment variable, so we need to get it as a string anyway, what do you think?

File: commands/scanpullrequest.go [hunk 3]
```diff
@@ -73,6 +79,21 @@ func createVulnerabilitiesRows(previousScan, currentScan []services.ScanResponse
 	return vulnerabilitiesRows
```
[the remark this answers is not in the document]
Comment (omerzi) [reply commands/scanpullrequest.go:79]: done

File: commands/scanpullrequest.go [hunk 4]
```diff
@@ -39,17 +39,23 @@ func scanPullRequest(params *utils.FrogbotParams, client vcsclient.VcsClient) er
 	if err != nil {
 		return err
 	}
-
-	// Audit target code
-	previousScan, err := auditTarget(client, xrayScanParams, params)
-	if err != nil {
-		return err
+	var vulnerabilitiesRows []formats.VulnerabilityOrViolationRow
+	if params.IncludeAllVulnerabilities == true {
```
Comment (talarian1) [inline commands/scanpullrequest.go:43]:
```suggestion
	if params.IncludeAllVulnerabilities {
```

File: commands/utils/params.go [hunk 5]
```diff
@@ -120,6 +121,11 @@ func extractGitParamsFromEnv(params *FrogbotParams) error {
 
 func extractGeneralParamsFromEnv(params *FrogbotParams) {
 	params.WorkingDirectory = getTrimmedEnv(WorkingDirectoryEnv)
+	var err error
+	params.IncludeAllVulnerabilities, err = strconv.ParseBool(getTrimmedEnv(IncludeAllVulnerabilitiesEnv))
+	if err != nil {
+		params.IncludeAllVulnerabilities = false
```
Comment (talarian1) [inline commands/utils/params.go:127]: Let's log the error, so customers can understand what went wrong with the value of this flag.

**Review by talarian1 — CHANGES_REQUESTED** [review]

**Review by talarian1 — APPROVED** [review]

**Pull request merged**